examples/storybook/src/stories/CurrentRefinements.stories.ts

```diff
@@ -0,0 +1,55 @@
+import { storiesOf } from '@storybook/angular';
+import { wrapWithHits } from '../wrap-with-hits';
+import meta from '../meta';
+
+storiesOf('CurrentRefinements', module)
+  .addDecorator(meta)
+  .add('default', () => ({
+    component: wrapWithHits({
+      template: '<ais-current-refinements></ais-current-refinements>',
+      searchParameters: {
```

Comment:
With a refinement widget it would be easier to test (see comment about `wrap-with-hits`).

Reply:
done! d06c4a1